### DISCUSSION TITLE
[@types/three] Commented out load and loadAsync in Loader.d.ts

### DISCUSSION THREAD

**Discussion by MTheProgrammer** [DefinitelyTyped/DefinitelyTyped#66341] [Issues with a @types package]
Hello, I have noticed that functions load and loadAsync are commented out in this interface:
https://github.com/DefinitelyTyped/DefinitelyTyped/blame/master/types/three/src/loaders/Loader.d.ts

They are present in the official Three.js documentation
https://threejs.org/docs/#api/en/loaders/Loader.load
https://threejs.org/docs/#api/en/loaders/Loader.loadAsync

Right now it is not possible to write tests without hacking the Loader mock.

Do you know why these functions were not implemented?

[a comment by an automated account is not shown]

**Comment by Methuselah96** [comment]
This should be fixed by https://github.com/three-types/three-ts-types/pull/570, which will be released with @types/three@0.156.0.
